.coveragerc

```diff
@@ -1,5 +0,0 @@
-[report]
```

Comment:
removed this config file since there is no service attached to the repo that requires this